lib/pliny/error_reporters/rollbar.rb

```diff
@@ -18,7 +18,7 @@ def notify(exception, context:, rack_env:)
       private
 
       def fetch_scope(context:, rack_env:)
-        scope = {}
+        scope = { custom: context }
```

Comment:
It does hold significance to rollbar. Outside of the request object, they require data be under a `custom` key. See https://rollbar.com/docs/api/items_post/.